src/main/java/seedu/address/model/exam/DistinctExamList.java

```diff
@@ -41,6 +41,7 @@ public boolean contains(Exam toCheck) {
      */
     public boolean containsModule(Module toCheck) {
         requireNonNull(toCheck);
+
         return examList.stream().map(Exam::getModule).anyMatch(toCheck::isSameModuleCode);
```

Comment:
I think this is the line that is throwing the error cos isSameModuleCode has been removed and changed to isSameModule